designer/server/src/views/forms/library.njk

```diff
@@ -27,16 +27,16 @@
     {% set formRowsMobile = [] %}
     {% set formRowsDesktop = [] %}
 
-    {% for formItem in formItems %}
+    {% for form in formItems %}
       {% set formName %}
-        <a href="{{ formItem.href }}" class="govuk-link govuk-link--no-visited-state">
-          {{ formItem.title }}
+        <a href="/editor/{{ form.id | urlencode }}" class="govuk-link govuk-link--no-visited-state">
+          {{ form.title }}
         </a>
       {% endset %}
 
       {% set formUpdated %}
-        <span class="app-display-until-desktop">Updated </span>{{ formItem.updated.date }}<br>
-        by {{ formItem.updated.name }}
+        <span class="app-display-until-desktop">Updated </span>{{ (form.draft.updatedAt if form.draft.updatedAt else "2024-04-09") | formatDate }}<br>
+        by {{ form.draft.updatedBy if form.draft.updatedBy else "Enrique Chase" }}
```

Comment:
`form.draft.updatedBy | default("Enrique Chase")` is a little more succinct

Rather than a dummy name I'd just "unknown user" though - might as well make it more presentable.

Reply:
I've kept good old Enrique 😆

He's just been [moved from the view model](https://github.com/DEFRA/forms-designer/pull/143/commits/f11964c2bafa2ac97ae1852ee7dd08be02832f5e#diff-df21d4f3e8fd1eb0ab49dac7e9a316f79e23a6c3214e7cfb79ce1011551de7e8L13) for now

Hopefully we're not too far off filling these gaps